tremor-cli/src/doc.rs

```diff
@@ -0,0 +1,149 @@
+// Copyright 2018-2020, Wayfair GmbH
```

Comment:
I was thinking of adding a `tremor doc` command for that - we can use the `cli.yaml` clap file and some extra text and examples to generate the docs?

Reply:
ah yes that too I was thinkinga bout the generated docs for our function library, they currently use tremot-tool doc we got to update that, I put a note here because I'll have forgotten tomorrow :joy:

Reply:
Let's brainstorm a snag list and add to the PR ...